src/alarms/notifications/AlarmRow.js

```diff
@@ -13,64 +13,6 @@ class AlarmRow extends Component {
       isFetching: true,
       isActive: props.alarm.active
     };
-    this.activateAlarm = this.activateAlarm.bind(this);
-    this.deActivateAlarm = this.deActivateAlarm.bind(this);
-  }
-
-  activateAlarm(uuid) {
-    const { addNotification } = this.props;
-
-    fetch(`/api/v3/rasteralarms/${uuid}/`, {
-      credentials: "same-origin",
-      method: "PATCH",
-      headers: { "Content-Type": "application/json" },
-      body: JSON.stringify({
-        active: true
-      })
-    })
-      .then(response => response.json())
-      .then(data => {
-        this.setState({
-          isActive: true
-        });
-        addNotification(`Alarm activated`, 2000);
-      });
-  }
-
-  deActivateAlarm(uuid) {
-    const { addNotification } = this.props;
-
-    fetch(`/api/v3/rasteralarms/${uuid}/`, {
-      credentials: "same-origin",
-      method: "PATCH",
-      headers: { "Content-Type": "application/json" },
-      body: JSON.stringify({
-        active: false
-      })
-    })
-      .then(response => response.json())
-      .then(data => {
-        this.setState({
-          isActive: false
-        });
-        addNotification(`Alarm deactivated`, 2000);
-      });
-  }
-
-  removeAlarm(uuid) {
-    const { loadAlarmsOnPage, addNotification } = this.props;
-
-    fetch(`/api/v3/rasteralarms/${uuid}/`, {
-      credentials: "same-origin",
-      method: "DELETE",
-      headers: { "Content-Type": "application/json" },
-      body: JSON.stringify({})
-    }).then(response => {
-      if (response.status === 204) {
-        loadAlarmsOnPage(1);
-        addNotification(`Alarm removed`, 2000);
-      }
-    });
   }
```

Comment:
Dit zou dan naar Redux actions gaan, niet op een component. Zowel dingen op 'fetching' zetten als het ontvangen van de response hebben effecten in Redux.

Reply:
Besloten om deze acties voor nu niet in redux te doen.
Het merendeel van deze acties verlopen momenteel niet via redux.
We kunnen dit later deze sprint alsnog doen.